waiter/src/waiter/core.clj

```diff
@@ -700,8 +700,12 @@
    :waiter-request?-fn (pc/fnk [[:settings hostname]]
                          (let [local-router (InetAddress/getLocalHost)
                                waiter-router-hostname (.getCanonicalHostName local-router)
-                               waiter-router-ip (.getHostAddress local-router)]
-                           (waiter-request?-factory (set [hostname waiter-router-hostname waiter-router-ip]))))
+                               waiter-router-ip (.getHostAddress local-router)
+                               hostnames (if (sequential? hostname)
+                                           (clojure.set/union (set hostname)
```

Comment:
`clojure.set` is already imported, we can just do `set/union`

Reply:
Done